<!DOCTYPE HTML>
<!--
	Arcana by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Systems Biology for Oncology - Federica Eduati group</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">
		<div id="page-wrapper">

			<!-- Header -->
				<div id="header">

					<!-- Logo -->
						<h1><a href="index.html" id="logo">Systems Biology for Oncology - Eduati lab</a></h1>
						<header>
							</header>

					<!-- Nav -->
						<nav id="nav">
							<ul>
								<li class="current"><a href="index.html">Home</a></li>
								<li><a href="#research">Research</a></li>
								<li><a href="#publications">Publications</a></li>
								<li><a href="#team">Team</a></li>
								<li><a href="#contact">Contact</a></li>
							</ul>
						</nav>

				</div>

			<!-- Banner -->
				<section id="banner">
					<header>
<a href="https://www.tue.nl/en/research/research-groups/computational-biology/">Computational Biology group</a> (Department of Biomedical Engineering)<br/>
<a href="https://www.tue.nl/en/research/institutes/institute-for-complex-molecular-systems/">Institute for Complex Molecular Systems</a> at Eindhoven University of Technology.</p>
					</header>
				</section>

			<!-- Research -->
				<section id="research" section class="wrapper style1">
					<div class="container">
						<header class="major">
							<h2>Research</h2>
							<p>The response to therapy shows great variation among cancer patients, even when patients are diagnosed with the same type of cancer. The main reason is that no two tumors are the same; tumors are composed of many different cell types and the specific composition and functioning of a tumor affects the response to treatment. By viewing tumors as complex systems, the use of mathematical models can help to better describe and understand the relationship between composition of the tumor and its behavior. Using clinical data and knowledge of cancer biology, we develop such models to study what determines a tumor's response to treatment. These insights may help to improve cancer drugs and increase the chances of the positive response to treatment.<br/><br/>
							<strong>In our group we take a multidisciplinary and holistic approach to provide a functional understanding of drugs response in the tumor microenvironment to improve precision oncology.</strong>
							</p>
						<a href="#" class="image"><img src="images/SystemsBiologyApproach.jpeg" alt="" width="400" /></a>
						</header>
						<header>
							<h3>Predictive biomarkers from functional analysis of static data</h3>
							<p>
								<ul>
									<li>Integration of static bulk RNAseq data with different sources of prior knowledge to derive a quantitative characterisation of different facets of the TME that is predictive of response to immune checkpoint inhibitors. Tool: <a href="https://bioconductor.org/packages/release/bioc/html/easier.html">EaSIeR</a>. Paper: <a href="https://doi.org/10.1016/j.patter.2021.100293">Lapuente-Santana et al, Patterns, 2021</a>).</li>
									<li>Inference of personalised (ensembles of) cell-cell interaction networks from bulk RNA-seq data and prior knowledge using random graph models. Tool: <a href="https://github.com/SysBioOncology/RaCInG">RaCInG</a>. Paper: <a href="https://www.biorxiv.org/content/10.1101/2023.02.03.526946v1">van Santvoort et al, bioRxiv, 2023</a>).</li>
								</ul>
							</p>
							<h3>Spatial multicellular models</h3>
							<p>
								<ul>
									<li>Development of computational approaches to quantitatively characterise spatial localization of cell types in the tumor microenvironment using radiogenomics approaches, i.e., combining imaging features and gene expression. </li>
									<li>Development of agent-based models to study how interactions between different cells in the TME coordinate tumour development and response to treatment. Models: <a href="https://github.com/SysBioOncology/ABM_prostate_cancer_development">Prostate cancer development</a>, <a href="https://github.com/SysBioOncology/PCABM_ADT">Prostate cancer androgen deprivation therapy</a>. Papers: <a href="https://aacrjournals.org/cancerrescommun/article/doi/10.1158/2767-9764.CRC-23-0097/727779">Passier et al, Cancer Research Communications, 2023</a>, <a href="https://www.biorxiv.org/content/10.1101/2023.07.27.550792v1.abstract">van Genderen et al, bioRxiv, 2023</a>).</li>
								</ul>
							</p>
							<h3>Diagnostic and prognostic biomarkers from liquid biopsies</h3>
							<p>
								<ul>
									<li>Development of machine learning models to use tumour markers (TMs) from liquid biopsies such as circulating tumour DNA (ctDNA) and proteins in the blood for diagnosis and monitoring of lung cancer. Model: <a href="https://github.com/SysBioOncology/Lungmarkerstudy_diagnosticalgorithm">Diagnostic lung cancer model</a>. Paper: <a href="https://doi.org/10.1016/j.lungcan.2023.01.014">Visser et al, Lung cancer, 2023</a>).</li>
								</ul>
							</p>
							<h3>Mechanistic models from functional perturbation data</h3>
							<p>
								<ul>
									<li>Development of microfluidics technology for high-throughput perturbation screening of live cells from cancer patient biopsies. Paper: <a href="https://www.nature.com/articles/s41467-018-04919-w">Eduati et al, Nature Communications, 2018</a>).</li>
									<li>Inference predictive personalised mathematical models of intracellular signalling pathways based on logic ordinary differential equations to understand cellular mechanisms mediating differential drug response in patients. Paper: <a href="https://www.embopress.org/doi/full/10.15252/msb.20188664">Eduati et al, Molecular Systems Biology, 2020</a>).</li>
								</ul>
							</p>
						</header>
					</div>
				</section>
				
			<!-- Publications -->
				<section id="publications" section class="wrapper style2">
					<div class="container">
						<header class="major">
						<h2>Selected publications</h2>
						<a href="https://scholar.google.com/citations?user=SAeklM8AAAAJ&hl=en&oi=ao">Full list of publications in Google Scholar.</a>
						</header>
						<header>
							<h4><a href="https://www.biorxiv.org/content/10.1101/2023.07.27.550792v1">Exploring the onset and progression of prostate cancer through a multicellular agent-based model</a></h4>
							<p style="font-size:16px; ">M. van Genderen<sup>$</sup>, J. Kneppers<sup>$</sup>, A. Zaalberg<sup>$</sup>, E. Bekers, A. M. Bergman<sup>#</sup>, W. Zwart<sup>#</sup>, F. Eduati<sup>#</sup>. <i>bioRxiv</i>. , 2023. (<sup>$</sup> co-first authors, <sup>#</sup> co-last authors)</p>
							<h4><a href="https://www.biorxiv.org/content/10.1101/2023.02.03.526946v1">Mathematically mapping the network of cells in the tumor microenvironment</a></h4>
							<p style="font-size:16px; ">M. van Santvoort, Ó. Lapuente-Santana, F. Finotello, P. van der Hoorn<sup>#</sup>, F. Eduati<sup>#</sup>. <i>bioRxiv</i>. 2023. (<sup>#</sup> co-last authors)</p>
							<h4><a href="https://www.biorxiv.org/content/10.1101/2023.02.16.528831v1">Exploring the onset and progression of prostate cancer through a multicellular agent-based model</a></h4>
							<p style="font-size:16px; ">M. Passier, M. van Genderen, A. Zaalberg, J. Kneppers, E. Bekers, A. M. Bergman<sup>#</sup>, W. Zwart<sup>#</sup>, F. Eduati<sup>#</sup>. <i>Cancer Research Communications</i>. CRC-23-0097, 2023. (<sup>#</sup> co-last authors)</p>
							<h4><a href="https://linkinghub.elsevier.com/retrieve/pii/S0169500223000442">Liquid biopsy-based decision support algorithms for diagnosis and subtyping of lung cancer</a></h4>
							<p style="font-size:16px; ">E. Visser<sup>$</sup>, S. A.A.M. Genet<sup>$</sup>, R. P.P.A. de Kock, B. E.E.M. van den Borne, M. Youssef-El Soud, H. N.A. Belderbos, G. Stege, M. E.A. de Saegher, S. C. van 't Westeinde, L. Brunsveld, M. A. C. Broeren, D. van de Kerkhof, B. A.L.M. Deiman, F. Eduati<sup>#</sup>, V. Scharnhorst<sup>#</sup>. <i>Lung Cancer</i>. 178, 28-36, 2023. (<sup>$</sup> co-first authors, <sup>#</sup> co-last authors)</p>
							<h4><a href="https://www.cell.com/trends/cell-biology/fulltext/S0962-8924(22)00109-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS096289242200109X%3Fshowall%3Dtrue">Technological and computational advances driving high-throughput oncology</a></h4>
							<p style="font-size:16px; ">L. Kolmar<sup>$</sup>, A. Autour<sup>$</sup>, X. Ma, B. Vergier, F. Eduati<sup>#</sup>, C. A. Merten<sup>#</sup>. <i>Trends in Cell Biology</i>, 32(11), 847-961, 2022. (<sup>$</sup> co-first authors, <sup>#</sup> co-last authors)</p>
							<h4><a href="https://doi.org/10.1016/j.patter.2021.100293">Interpretable systems biomarkers predict response to immune-checkpoint inhibitors</a></h4>
							<p style="font-size:16px; ">Ó. Lapuente-Santana, M. van Genderen, P. A. J. Hilbers, F. Finotello, F. Eduati. <i>Patterns</i>, 2(8), 100293. 2021. </p>
							<h4><a href="https://www.frontiersin.org/articles/10.3389/fonc.2020.01027/fulle">Toward Systems Biomarkers of Response to Immune Checkpoint Blockers</a></h4>
							<p style="font-size:16px; ">Ó. Lapuente-Santana, F. Eduati. <i>Frontiers in Oncology</i>, 10:1027. 2020.</p>
							<h4><a href="https://www.embopress.org/doi/full/10.15252/msb.20188664">Patient-specific logic models of signaling pathways from screenings on cancer biopsies to prioritize personalized combination therapies</a></h4>
							<p style="font-size:16px; ">F. Eduati, P. Jaaks, J. Wappler, T. Cramer, C. A. Merten, M. J. Garnett, J. Saez-Rodriguez. <i>Molecular Systems Biology</i>, 16(2),e8664. 2020.</p>
							<h4><a href="https://www.nature.com/articles/s41467-018-04919-w/">A microfluidics platform for combinatorial drug screening on cancer biopsies.</a></h4>
							<p style="font-size:16px; ">F. Eduati<sup>$</sup>, R. Utharala<sup>$</sup>, D. Madhavan, U.P. Neumann, T. Longerich, T. Cramer, J. Saez- Rodriguez<sup>#</sup>., C. Merten<sup>#</sup>. <i>Nature Communications</i>, 9(1):2434, 2018. (<sup>$</sup> co-first authors, <sup>#</sup> co-last authors) </p>
							<h4><a href="https://www.frontiersin.org/articles/10.3389/fonc.2018.00430/full">Multi-Omics Profiling of the Tumor Microenvironment: Paving the Way to Precision Immuno-Oncology. </a></h4>
							<p style="font-size:16px; ">F. Finotello, F. Eduati. <i>Frontiers in Oncology</i>, 8, 1–9. 2018.</p>
							<h4><a href="https://doi.org/10.1158/0008-5472.CAN-17-0078">Drug resistance mechanisms in colorectal cancer dissected with cell type-specific dynamic logic models.</a></h4>
							<p style="font-size:16px; ">F. Eduati, V. Doldàn-Martelli, B. Klinger, T. Cokelaer, A. Sieber, F. Kogera, M. Dorel, M. J. Garnett, N. Blüthgen<sup>#</sup>, J. Saez-Rodriguez<sup>#</sup>. <i>Cancer Research</i>, 77(12):3364-3375, 2017. (<sup>#</sup> co-last authors) </p>
							<h4><a href="https://www.nature.com/articles/nbt.3299">Prediction of human population responses to toxic compounds by a collaborative competition.</a></h4>
							<p style="font-size:16px; ">F. Eduati<sup>$</sup>, L. M. Mangravite<sup>$</sup>, T. Wang*, H. Tang*, J. C. Bare, R. Huang, T. Norman, M. Kellen, M. P. Menden, J. Yang, X. Zhan, R. Zhong, G. Xiao, M. Xia, N. Abdo, O. Kosyk, The NIEHS-NCATS-UNC DREAM Collaboration, S. Friend, A. Dearry, A. Simeonov, R. R. Tice, I. Rusyn, F. A. Wright, G. Stolovitzky, Y. Xie, and J. Saez-Rodriguez. <i>Nature Biotechnology</i>, 733(9):933-940, 2015. (<sup>$</sup> co-first authors) </p>
						</header>
					</div>
				</section>
				

			
				
			<!-- Team -->	
				<section id="team" class="wrapper style1">
					<header class="major">
					<h2>Team members</h2>
					</header>
					<div class="container">	
						<h2>Group Leader</h2>
						<div class="row">
								<div class="col-3">
									<a href="#" class="image left"><img src="images/Federica.png" alt="" /></a>
								</div>
								<div class="col-7">
										<h3>Federica Eduati</h3>
										<p>Since 2018 Assistant professor, Eindhoven University of Technology, NL<br>
										   2013-2017 Postdoc, European Molecular Biology Laboratory (EMBL), DE & UK<br>
										   2013 PhD in Bioengineering, University of Padova, IT<br>
										   2008 MSc in Bioengineering, University of Padova, IT<br>
										</p>
										<p>Research interests: Systems approaches to precision and personalised medicine in oncology. <a href="https://www.tue.nl/en/research/researchers/federica-eduati">Personal page at TU/e.</a></p>
										
								</div>
								
						</div>
					</div>
					
					<div class="container">	
						<h2>Postdoc</h2>
						<div class="row">
								<div class="col-3">
									<a href="#" class="image left"><img src="images/Maaruthy.png" alt="" /></a>
								</div>
								<div class="col-7">
										<h3>Maaruthy Yelleswarapu</h3>
										<p>Since 2018 Postdoc, Eindhoven University of Technology, NL<br>
										   2019-2021 Postdoc, Rijksuniversiteit Groeningen, NL<br>
										   2019 PhD in Physical Organic Chemistry, Radboud University, NL<br>
										   2012 MSc in Biotechnology, ETH Zurich, CH<br>
										</p>
										<p>Research project: Developing a microfluidic platform for the combinatorial screening of cancer biopsies to elucidate tumor-mediated response to drugs.</p>
								</div>
								
						</div>
					</div>
					
					<div class="container">	
						<h2>PhD students</h2>
						<div class="row">
								<div class="col-3">
									<a href="#" class="image left"><img src="images/Esther.png" alt="" /></a>
								</div>
								<div class="col-7">
										<h3>Esther Visser</h3>
										<p>Since 2020 PhD student, Catharina Hospital & Eindhoven University of Technology, NL<br>
										   2020 MSc in Medical Engineering, Eindhoven University of Technology, NL<br>
										</p>
										<p>Research project: Analyzing the clinical applicability of tumor markers in the diagnosis and treatment response evaluation of lung cancer patients.</p>
								</div>
								
						</div>
						<div class="row">
								<div class="col-3">
									<a href="#" class="image left"><img src="images/Mike.png" alt="" /></a>
								</div>
								<div class="col-7">
										<h3>Mike van Santvoort</h3>
										<p>Since 2021 PhD student, Eindhoven University of Technology, NL<br>
										   2020 MSc in Industrial and Applied Mathematics, Eindhoven University of Technology, NL<br>
										</p>
										<p>Research project: Designing and analyzing random graph models for reconstruction of cell-cell interaction networks to understand differences in and between tumors.</p>
								</div>
								
						</div>
						<div class="row">
								<div class="col-3">
									<a href="#" class="image left"><img src="images/tmp.png" alt="" /></a>
								</div>
								<div class="col-7">
										<h3>Glenn Weber</h3>
										<p>Since 2023 PhD student, Eindhoven University of Technology, NL<br>
										   2022 MSc in Systems Biology, Maastricht University, NL<br>
										</p>
										<p>Research project: .</p>
								</div>
								
						</div>
						<div class="row">
								<div class="col-3">
									<a href="#" class="image left"><img src="images/Hao.png" alt="" /></a>
								</div>
								<div class="col-7">
										<h3>Hao Cao</h3>
										<p>Since 2023 PhD student, Catharina Hospital & Eindhoven University of Technology, NL<br>
										   2022 MSc in Medical Engineering, Eindhoven University of Technology, NL<br>
										</p>
										<p>Research project: Implementation of up-front ctDNA analysis into lung cancer care and development of liquid biopsy-based decision support models.</p>
								</div>
								
						</div>
					</div>
					
					<div class="container">	
						<h2>MS students</h2>
						<h3>Livy Nijhuis</h3>
						<p>Research project: Inferring cell-cell interaction networks from spatial transcriptomics data.</p>
						<h3>Jojanne van Leeuwen</h3>
						<p>Research project: Modeling therapy response in prostate cancer using an agent-based model.</p>
						<h3>Jolien Marcelis</h3>
						<p>Research project: Studying the effect of anticancer drugs on PD-L1 expression.</p>
						<h3>Bi-rong Wang</h3>
						<p>Research project: Using cytokine-beads microfluidics for reconstructing signaling pathways in tumor cells.</p>
						<h3>Eline Vos</h3>
						<p>Research project: TBD</p>
						
					</div>
					
						
				</section>
				
				<section id="team" class="wrapper style1">
					<header class="major">
					<h2>Alumni</h2>
					</header>
					<div class="container">	
						<head>
						<style>
						table {
						  border-collapse: collapse;
						  width: 100%;
						}

						th, td {
						  padding: 8px;
						  text-align: left;
						  border-bottom: 1px solid #DDD;
						}

						tr:hover {background-color: #D6EEEE;}
						</style>
						</head>
						<body>
						
						<table style="width:100%">
						  <tr>
						    <th><b>Year(s)</b></th>
						    <th><b>Name</b></th>
						    <th><b>Position</b></th>
						  </tr>
						  <tr>
						    <td>2018-2023</td>
						    <td>Óscar Lapuente-Santana</td>
						    <td>Erasmus student (2018), PhD student (2019-2023)</td>
						  </tr>
						  <tr>
						    <td>2022</td>
						    <td>Sofia Spinthaki</td>
						    <td>Internship (Erasmus+)</td>
						  </tr>
						  <tr>
						    <td>2021-2022</td>
						    <td>Joan Kant</td>
						    <td>Master thesis</td>
						  </tr>
						  <tr>
						    <td>2021-2022</td>
						    <td>Max Joosten</td>
						    <td>Master thesis</td>
						  </tr>
						  <tr>
						    <td>2021-2022</td>
						    <td>Max de Rooij</td>
						    <td>Master thesis</td>
						  </tr>
						  <tr>
						    <td>2020-2021</td>
						    <td>Margot Passier</td>
						    <td>Master thesis</td>
						  </tr>
						  <tr>
						    <td>2020-2021</td>
						    <td>Roy van Mierlo</td>
						    <td>Internship</td>
						  </tr>
						  <tr>
						    <td>2019-2020</td>
						    <td>Maisa van Genderen</td>
						    <td>Master thesis</td>
						  </tr>
						  <tr>
						    <td>2018</td>
						    <td>Kees van Dorp</td>
						    <td>Internship</td>
						  </tr>
						</table>
					</div>
				</section>
				
				

			<!-- Footer -->
				<section id="contact" section class="wrapper style2">
										<div class="container">
											<h2 class="major">Contact</h2>

											<ul class="contact">
												<li class="icon solid fa-home">
													PO Box 513, 5600 MB Eindhoven, the Netherlands<br/>
													Visiting address: Building Gemini Zuid, 3.105<br/>
													Navigation: De Zaale, Eindhoven<br/>
													Postal and parcel deliveries: De Rondom 70, 5612 AP Eindhoven
												</li>
												<li class="icon solid fa-envelope"><a href="mailto:f.eduati@tue.nl">  Get in touch via e-mail</a></li>
												<li class="icon brands fa-twitter"><a href="https://twitter.com/FEduati">  Follow us on Twitter</a></li>
												<li class="icon brands fa-twitter"><a href="https://www.linkedin.com/in/eduati/">  Follow Federica on LinkedIn</a></li>
												<li class="icon brands fa-github"><a href="https://github.com/SysBioOncology">  Check out our GitHub</a></li>
											</ul>
											
										</div>
									</section>

					<!-- Copyright -->
						<div class="copyright">
							<ul class="menu">
								<li>&copy; Untitled. All rights reserved</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>

				</div>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>